lib/rules/decorator-position.js

```diff
@@ -137,17 +134,14 @@ function decoratorPositionRule(context) {
   }
 
   const userOptions = context.options[0] || {};
-  const filePath = context.getFilename();
-  const { printWidth } = lineLength(userOptions, filePath);
-  const options = normalizeOptions({ ...userOptions, printWidth });
+  const options = normalizeOptions(userOptions);
 
   return {
-    'ClassProperty[decorators.length=1]:exit'(node) {
```

Comment:
What advantage does this new selector with siblings have over the previous?

Reply:
As a result of experiments, I found out that there is no ClassProperty selector in eslint 8 anymore.